### DISCUSSION TITLE
Add DomProperty and DomAttribute to Condition Class

### DISCUSSION THREAD

**Discussion by saimanikandanv** [selenide/selenide#2782] [Ideas]
@asolntsev 
As Selenium 4 introduces new methods getDomProperty()and getDomAttribute(), expecting same to be added to selenide condition class for validations

**Comment by asolntsev** [comment]
@saimanikandanv @rosolko  Can you **provide some realistic examples** where methods `getDomProperty()` and `getDomAttribute()` would be useful in tests?

When I as reading [their description](https://saucelabs.com/resources/blog/selenium-4-new-element-attribute-and-property-methods), I had a feeling that they are not really needed. 

Let's assume there is 
1. Method `getDomProperty()` returns a fresh value for changed attribute, BUT "null" for unchanged attributes.
2. Method `getDomAttribute` returns the fresh attribute value, but not always (depends on element implementation).

While the good old method `getAttribute()` returns exactly what user needs.

**Reply by saimanikandanv** [reply]
I was working on react application put inside canvas, there was a field label field with some text, there are no attributes defined by the dev. Hence i used a html property isContentEditable property and am verifying whether it is false to ensure its readonly

Thought i have read the value using getDomAttribute() and asserted but i feel this might help somewhere. Hence opened this thread to get more insights from you on this...
